haystack/components/embedders/hugging_face_api_document_embedder.py

```diff
@@ -14,7 +14,7 @@
 from haystack.utils.hf import HFEmbeddingAPIType, HFModelType, check_valid_model
 from haystack.utils.url_validation import is_valid_http_url
 
-with LazyImport(message="Run 'pip install \"huggingface_hub>=0.23.0\"'") as huggingface_hub_import:
+with LazyImport(message="Run 'pip install \"huggingface_hub>=0.27.0\"'") as huggingface_hub_import:
```

Comment:
This is only a suggestion for users.
The `ChatCompletionInputTool` type was introduced recently, so it is better to install the latest version.